daemon/oci_linux.go

```diff
@@ -803,7 +803,10 @@ func WithDevices(daemon *Daemon, c *container.Container) coci.SpecOpts {
 		// Build lists of devices allowed and created within the container.
 		var devs []specs.LinuxDevice
 		devPermissions := s.Linux.Resources.Devices
-		if c.HostConfig.Privileged && !rsystem.RunningInUserNS() {
+		if c.PrivilegedWithoutHostDevices && !c.HostConfig.Privileged {
+			return errors.New("privileged-without-host-devices requires privileged mode to be enabled")
```

Comment:
@AkihiroSuda  Where are we actually restricting the host devices or `/dev` from getting mounted inside the container? I think this just errors out right??

Reply:
the next line